stratify/_vinterp.pyx

```diff
@@ -544,10 +543,6 @@ cdef class _Interpolation(object):
         fz_src = fz_src.astype(np.float64)
 
         # Broadcast the z_target shape if it is 1d (which it is in most cases)
-        if z_target.ndim == 1:
-            z_target_size = z_target.shape[0]
-        else:
-            z_target_size = z_target.shape[axis]
```

Comment:
I agree the variable names take a bit of getting used to .... I had to loop over it quite a few times before the name of the thing connected with what it was.

If we go for a rebrand of variable names, then I wouldn't want to do it in this PR. That would be crazy noisy. So I'd push back to doing that in a separate PR, is that okay @cpelley ?